vividus-plugin-applitools/src/main/java/org/vividus/bdd/converter/ExamplesTableToApplitoolsVisualChecksConverter.java

```diff
@@ -20,15 +20,12 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-import javax.inject.Named;
-
 import org.apache.commons.lang3.Validate;
 import org.jbehave.core.model.ExamplesTable;
 import org.jbehave.core.steps.ParameterConverters.AbstractChainableParameterConverter;
 import org.vividus.visual.eyes.factory.ApplitoolsVisualCheckFactory;
 import org.vividus.visual.eyes.model.ApplitoolsVisualCheck;
 
-@Named
```

Comment:
why it is necessary?

Reply:
due to @Named converters are loaded regardless of the current profile, so when we start up mobile_app profile this converter is registered as a bean, but since it has dependencies on beans from web profile, the spring context startup fails